wp-theme-2018/sidebar.php

```diff
@@ -54,15 +86,48 @@
         <aside class="nav-aside-wrapper">
             <nav id="nav-aside" class="nav-aside" role="navigation" aria-describedby="nav-aside-title">
                 <h2 class="h5 sr-only-xl"><?php esc_html_e("In the same section", 'epfl') ?></h2>
-                <?php
-                wp_nav_menu( array(
-                    'theme_location' => $EPFL_MENU_LOCATION,
-                    'menu_class'=> 'nav-menu',
-                    'container' => 'ul',
-                    'submenu' => get_the_ID(),
-                    'submenu_type' => $asideContent
-                ) );
-                ?>
+                <ul id="menu-main" class="nav-menu">
+                    <?php
+                        $items = get_all_menu_items_flattened();
+
+                        $current_item = get_current_item($items);
+
+                        // fullfil crumb_items array, in accordance with the items hierarchy
+                        $crumb_items = [];
```

Comment:
I don't see `$crumb_items` being used for reading anywhere in this file? Please clean up the dead code.